spec/filters/grok_spec.rb

```diff
@@ -1,5 +1,13 @@
 # encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+
+# running the grok code outside a logstash package means
+# LOGSTASH_HOME will not be defined, so let's set it here
+# before requiring the grok filter
+unless LogStash::Environment.const_defined?(:LOGSTASH_HOME)
```

Comment:
You're going to hate me, but why not having this code add in a common helper file ? 

we could have something like 

```
  require_relative "../spec_helper"
```

and then add this fix in this file, isn't?